_posts/2013-01-11-navigate-using-just-your-keyboard.md

```diff
@@ -9,11 +9,26 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+### Introduction
+Testing your site for keyboard navigation will ensure that keyboard users can access your site.
 
-1. Un-plug/turn off your mouse.
-2. Use the <kbd>Tab</kbd> key browse.
-3. Use <kbd>Enter</kbd> to select elements.
-4. Some menus and form elements may require <kbd>&uarr;</kbd> <kbd>&darr;</kbd> <kbd>&larr;</kbd> <kbd>&rarr;</kbd> arrow keys.
+As a bonus, sites that can be navigated with a keyboard can also likely be navigated using other input devices.
```

Comment:
"can also likely" is an affirmative followed by a maybe.

"are more likely to be navigable"

would be safer a wording.